script/Arcueid/Day 6/06_00_ARC06_2_2.txt

```diff
@@ -410,18 +411,19 @@ You may look docile, but deep down you must be cold-blooded, right?
 [sha:d52425c4da1b2367075d7ce4250da6f8b911650f]{
 -- Page 243, Offset 14501.
 -- 「筋肉には瞬発力重視の速筋と、持久力重視の遅筋があるワケだけど、志貴チャンは速筋メイン。それはいいんだけどぉ……なんていうか、軽いのよねぇ。骨格じゃなくて骨そのものが軽いというか。なんか、生まれつき自重をごまかしてふわふわ浮く不思議チャン？みたいな？」
+// to be rechecked
 "When it comes to muscle, there's the fast-twitch variety, which is specialized for brief periods of explosive power,
- and the slow-twitch variety, which is better suited for long term endurance.
+ and the slow-twitch variety, which is better suited for long-term endurance.
  You're mostly made up of the former.
  That's all well and good, but, how do I put this... it's light.
- It's not as though your bone structure is light, more like the bones themselves are?
+ It's not as though you can have a light build, it's more like your bones themselves are?
```

Comment:
It's not as though you have a light build, but more like your bones themselves are light?